### PR TITLE
Add application template develop 

### PR DESCRIPTION
**Pull request by yunkunrao**

### CODE REVIEW

File: content/en/docs/application-store/app-lifecycle-management.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+title: "App Lifecycle Management"
+keywords: 'kubernetes, kubesphere, app-store'
+description: 'App Lifecycle Management'
+
+
+weight: 2240
+---
+
+KubeSphere integrates open source [OpenPitrix](https://github.com/openpitrix/openpitrix) to set up app store and app repository services which provide full lifecycle of application management. Application Store supports three kinds of application deployment as follows:
```
Comment (zheng1) [inline content/en/docs/application-store/app-lifecycle-management.md:10]: `KubeSphere integrates open-source project [OpenPitrix](https://github.com/openpitrix/openpitrix) to set up the App Store and which provide the full lifecycle of application management. App Store supports two kinds of application deployment as follows:`

File: content/en/docs/application-store/app-lifecycle-management.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+![App Store](/images/application-templates/app-store.png)
+
+## Objective
+
+In this tutorial, we will walk you through how to use [EMQ X](https://www.emqx.io/) as a demo application to demonstrate the **global application store** and **application lifecycle management** including upload / submit / review / test / release / upgrade / delete application templates.
+
+## Prerequisites
+
+- You need to install [Application Store (OpenPitrix)](../../installation/install-openpitrix).
```
Comment (zheng1) [inline content/en/docs/application-store/app-lifecycle-management.md:23]: `App Store`

File: content/en/docs/application-store/app-lifecycle-management.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+- You need to create a workspace and a project, see [Get Started with Multi-tenant Management](../admin-quick-start).
```
Comment (zheng1) [inline content/en/docs/application-store/app-lifecycle-management.md:24]: Both links do not open properly and need to be modified or replaced with a different text description

File: content/en/docs/application-store/app-lifecycle-management.md (hunk 2 above)
Comment (yunkunrao) [reply content/en/docs/application-store/app-lifecycle-management.md:23]: got it

File: content/en/docs/application-store/app-lifecycle-management.md (hunk 1 above)
Comment (yunkunrao) [reply content/en/docs/application-store/app-lifecycle-management.md:10]: got it

File: content/en/docs/application-store/app-lifecycle-management.md (hunk 3 above)
Comment (yunkunrao) [reply content/en/docs/application-store/app-lifecycle-management.md:24]: fixed

**Comment by Sherlock113** [comment]
/lgtm
The process looks clear to me. Please merge this pr and I will refine wording and update images in it later.

**Review by rayzhou2017 — APPROVED** [review]
/lgtm

**Pull request merged**